gmsh_utils/gmsh_IO.py

```diff
@@ -400,7 +396,7 @@ def generate_extract_mesh(self, dims: int, mesh_name: str, mesh_output_dir: str,
         if open_gmsh_gui:
             gmsh.fltk.run()
 
-        gmsh.finalize()
+        self.finalize_gmsh()
 
     def extract_node_data(self, node_tags: npt.NDArray[np.int_],
```

Comment:
np.int_ does not work with the docstring. 
use np.int64

Reply:
Check this in the entire file

Reply:
will be solved in #16